integration/graphql-types.ts

```diff
@@ -126,3 +141,9 @@ export enum Working {
   Yes = "YES",
   No = "NO",
 }
+
+export type UnionProp = String | Boolean;
```

Comment:
Is this name going to be unique across the whole schema? Like if you add another field that has a different union type like `Author.unionProp = Int | String`, what will it's name be?

Reply:
Oh I get it, didn't see the schema change / forgot these weren't declared inline.

Reply:
As-is, it is only creating these (I believe?) from explicit union type definitions in the schema, thus the names will be unique. I'll check to confirm what happens with this anonymous union type...

Reply:
Yeah, I'm ~80% sure that you can't even do inline / anonymous union types in GQL, which is convenient insofar as it forces the user to give globally unique names which is exactly what you want. :-)

Reply:
Looks like you are correct:
![image](https://user-images.githubusercontent.com/24765506/98555334-d1b5d100-226f-11eb-8174-16d6ebc98f6f.png)

👍